### PR TITLE
Content validation: extend logger

### PR DESCRIPTION
**Pull request by LowEntropyBody**
Work item: https://ceapex.visualstudio.com/Engineering/_workitems/edit/115503

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/5064)

### CODE REVIEW

File: src/docfx/lib/markdown/MarkdownUtility.cs [hunk 1]
```diff
@@ -86,7 +86,13 @@ public static (List<Error> errors, MarkdownDocument ast) Parse(string content, M
 
         private static MarkdownPipeline CreateMarkdownPipeline()
         {
-            var markdownContext = new MarkdownContext(GetToken, LogWarning, LogError, ReadFile);
+            var markdownContext = new MarkdownContext(
```
Comment (yufeih) [inline src/docfx/lib/markdown/MarkdownUtility.cs]:
```suggestion
            var markdownContext = new MarkdownContext(
                GetToken, LogInfo, LogSuggestion, LogWarning, LogError, ReadFile)
```

`LogInfo` can print everything `code, message, origin, line` to the console using the format defined in `ErrorLog.cs` line 240

Comment (LowEntropyBody) [reply src/docfx/lib/markdown/MarkdownUtility.cs]: cool

File: src/docfx/lib/markdown/MarkdownUtility.cs [hunk 2]
```diff
@@ -136,6 +148,11 @@ private static string GetToken(string key)
             return t_status.Value.Peek().Context.TemplateEngine.GetToken(key);
         }
 
+        private static void LogInfo(string code, string message, MarkdownObject origin, int? line)
+        {
+            ErrorLog.ConsoleLog(ErrorLevel.Off, new Error(ErrorLevel.Off, code, message, origin.ToSourceInfo(line)));
```
Comment (yufeih) [inline src/docfx/lib/markdown/MarkdownUtility.cs]: oops, maybe this is enough
```suggestion
            Log.Write($"{code} {message}");
```

Comment (LowEntropyBody) [reply src/docfx/lib/markdown/MarkdownUtility.cs]: yep

**Pull request merged**